docs/plugins/gmusic.rst

```diff
@@ -59,3 +40,45 @@ example::
 
 For a list of all songs in your library, run ``beet gmusic-songs`` without any
 arguments.
+
+
+Configuration
+-------------
+To configure the plugin, make a ``gmusic:`` section in your configuration file.
+The available options are:
+
+- **email**: Your Google account email address.  
+  Default: none.
+- **password**: Password to your Google account. Required to query songs in
+  your collection.  
+  For accounts with 2-step-verification, an
+  `app password <https://support.google.com/accounts/answer/185833?hl=en>`__
+  will need to be generated. An app password for an account without
+  2-step-verification is not required but is recommended.  
+  Default: none.
+- **auto**: Set to ``yes`` to automatically upload new imports to Google Play
+  Music.  
+  Default: ``no``
+- **uploader_id**: Unique id as a MAC address, eg ``'00:11:22:33:AA:BB'``.
+  This option should be set before the maximum number of authorized devices is
+  reached.  
+  If provided, use the same id for all future runs on this, and other, beets
+  installations as to not reach the maximum number of authorized devices.  
+  Default: device's MAC address.
+- **device_id**: Unique device ID for authorized devices.
+  This option only needs to be set if you receive an `InvalidDeviceId`
+  exception. Below the exception will be a list of valid device IDs.  
+  Default: none.
```

Comment:
A proper list of config options is great! Let’s definitely keep this stuff here.